openc-json_schema.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new do |gem|
   gem.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "json-schema", "~> 2.6.0"
+  gem.add_dependency "json", "2.1"
```

Comment:
Would it make more sense for this to be "<= 2.1" so as not to conflict with other requirements down the chain?

Reply:
Done in 3bf4ea087cadf75eefb7938a1398dff41da01117